src/commands/appSettings/connectionSettings/durableTaskScheduler/DTSConnectionListStep.ts

```diff
@@ -38,30 +42,48 @@ export class DTSConnectionListStep<T extends IDTSConnectionWizardContext> extend
             buttons.push(connectCustomDTSButton);
         }
 
+        buttons.push(skipForNow);
+
         const message: string = localize('selectDTSConnection', 'Durable Functions needs to be configured to use a Durable Task Scheduler.');
         context.dtsConnectionType = (await context.ui.showWarningMessage(message, { modal: true }, ...buttons) as {
             title: string;
             data: ConnectionType;
         }).data;
-
-        context.telemetry.properties.dtsConnectionType = context.dtsConnectionType;
     }
 
     public shouldPrompt(context: T): boolean {
         return !context.dtsConnectionType;
     }
 
-    public async getSubWizard(context: T): Promise<IWizardOptions<T> | undefined> {
+    public async getSubWizard(context: T | IDTSAzureConnectionWizardContext): Promise<IWizardOptions<T> | undefined> {
         const promptSteps: AzureWizardPromptStep<T | IDTSAzureConnectionWizardContext>[] = [];
         const executeSteps: AzureWizardExecuteStep<T | IDTSAzureConnectionWizardContext>[] = [];
 
+        context.telemetry.properties.dtsConnectionType = context.dtsConnectionType;
+
         switch (context.dtsConnectionType) {
             case ConnectionType.Azure:
-                promptSteps.push(new DurableTaskSchedulerListStep());
-                executeSteps.push(new VerifyProvidersStep<IDTSAzureConnectionWizardContext>([DurableTaskProvider]));
+                if (!(context as IDTSAzureConnectionWizardContext).subscriptionId) {
+                    Object.assign(context, createSubscriptionContext(await subscriptionExperience(context, ext.rgApiV2.resources.azureResourceTreeDataProvider)));
+                }
+
+                // LocationListStep.addProviderForFiltering(context as unknown as ILocationWizardContext, IdentityProvider, UserAssignedIdentityResourceType);
```

Comment:
Need to update a package to use this line, but I don't feel like doing that as part of the refactor because then it'll force me to make some additional changes as well.  I'll upgrade the package and uncomment this line in a future PR.